modules/sdk/pages/overview.adoc

```diff
@@ -3,31 +3,31 @@
 [abstract]
 Couchbase offers numerous software development kits (SDKs) that provide programmatic access to Couchbase Server.
 
-xref:3.0@c-sdk::start-using-sdk.adoc[C SDK 3.0] ::
+xref:3.0@c-sdk:hello-world:start-using-sdk.adoc[C SDK 3.0] ::
 The Couchbase C SDK (`libcouchbase`) enables C and C++ programs to access a Couchbase Server cluster.
 The C SDK is also commonly used as a core dependency of SDKs written in other language to provide a common implementation and high performance.
 
-xref:3.0@dotnet-sdk::start-using-sdk.adoc[.NET SDK 3.0] ::
+xref:3.0@dotnet-sdk:hello-world:start-using-sdk.adoc[.NET SDK 3.0] ::
 The .NET SDK enables you to interact with a Couchbase Server cluster from the .NET Framework using C#.
 It offers both a traditional synchronous API and an asynchronous API based on the Task-based Asynchronous Pattern (TAP).
 
-xref:2.0@go-sdk::start-using-sdk.adoc[Go SDK 2.0] ::
+xref:2.0@go-sdk:hello-world:start-using-sdk.adoc[Go SDK 2.0] ::
```

Comment:
Just making sure this doesn't need to be 3.0

Reply:
Thanks Eric - it would be lovely to have consistency, but semantic versioning means that Go is 2.0 (and Scala is 1.)